vault/resource_jwt_auth_backend_role_test.go

```diff
@@ -675,5 +688,6 @@ resource "vault_jwt_auth_backend_role" "role" {
     department = "engineering-*-admin"
     sector = "7g"
   }
+  user_claim_json_pointer = false
```

Comment:
Might be a good idea test the handling of the default value with a test config that does not specify this field.

Reply:
Good idea. Done in [f9b1c00](https://github.com/hashicorp/terraform-provider-vault/pull/1478/commits/f9b1c004d509e226479b1b3d78b0d55de385fab0).